src/main/resources/locale_de.properties

```diff
@@ -210,32 +210,32 @@ command.repsettings.emojidebug.message.receiverLimit=Das Empfängerlimit wurde e
 command.repsettings.emojidebug.message.title=Emoji Erklärung
 command.repsettings.emojidebug.message.tooold=Die referenzierte Nachricht ist zu alt
 command.repsettings.emojidebug.message.true=Der Bot wird erklärende Emojis an Nachrichten hinzufügen.
-command.repsettings.info.description=Die aktuellen Guild Settings.
+command.repsettings.info.description=Die aktuellen Servereinstellungen.
 command.repsettings.info.message.answer.false=Bot wird nicht in Antworten suchen.
 command.repsettings.info.message.answer.true=Bot wird in Antworten suchen.
 command.repsettings.info.message.choose=Wähle die Einstellung, die du ändern möchtest.
-command.repsettings.info.message.embed.false=Der Bot sendet wird keine Bestätigungsnachrichten schicken.
-command.repsettings.info.message.embed.true=Der Bot sendet wird Bestätigungsnachrichten schicken.
-command.repsettings.info.message.fuzzy.false=Es wird keine grobe Suche durchgeführt, wenn ein Dankwort erkannt wird.
-command.repsettings.info.message.fuzzy.true=Eine grobe Suche wird durchgeführt, wenn ein Dankwort erkannt wird.
+command.repsettings.info.message.embed.false=Der Bot wird keine Bestätigungsnachrichten schicken.
+command.repsettings.info.message.embed.true=Der Bot wird Bestätigungsnachrichten schicken.
+command.repsettings.info.message.fuzzy.false=Es wird keine grobe Suche durchgeführt, wenn ein Dankeswort erkannt wird.
+command.repsettings.info.message.fuzzy.true=Eine grobe Suche wird durchgeführt, wenn ein Dankeswort erkannt wird.
 command.repsettings.info.message.mention.false=Bot wird nicht in Nachrichten mit Erwähnungen suchen.
 command.repsettings.info.message.mention.true=Bot wird in Nachrichten mit Erwähnungen suchen.
-command.repsettings.info.message.option.byanswer.description=Antworten für Reputation zulassen oder verweigern
+command.repsettings.info.message.option.byanswer.description=Antworten zur Reputationvergabe
 command.repsettings.info.message.option.byanswer.name=Reputation per Antwort
-command.repsettings.info.message.option.byembed.description=Einbettung der Anfrage zulassen oder verweigern
+command.repsettings.info.message.option.byembed.description=Einbettung zur Reputationvergabe
 command.repsettings.info.message.option.byembed.name=Reputation durch Bestätigungsnachricht
-command.repsettings.info.message.option.byfuzzy.description=Fuzzy-Matching für Reputation zulassen oder verweigern
-command.repsettings.info.message.option.byfuzzy.name=Reputation per Fuzzy-Suche
-command.repsettings.info.message.option.bymention.description=Erwähnungen für die Reputation zulassen oder verweigern
-command.repsettings.info.message.option.bymention.name=Reputation per Erwähnung
-command.repsettings.info.message.option.byreaction.description=Reaktionen für Reputation zulassen oder verweigern
-command.repsettings.info.message.option.byreaction.name=Reputation per Reaktion
+command.repsettings.info.message.option.byfuzzy.description=Unscharfer Vergleich zur Reputationvergabe
```

Comment:
"Unscharfer Vergleich" sounds weird

Reply:
How would you translate "Fuzzy Match"?

Reply:
I don't know, but as it compares by the names, maybe "Namensbasierter Vergleich"?

Reply:
Unscharfe Übereinstimmung.